### ISSUE TITLE
Verify ownership on beta.flathub.org

### ISSUE THREAD

**Issue by barthalion** [flathub/org.libretro.RetroArch#247]
Hi @RobLoach,

We're preparing for the launch of beta.flathub.org, and would like to see RetroArch verified as maintained by its developers. Currently, I've marked it as such manually, but it would be preferred to use one of the verification features. This can be initiated by logging it at https://beta.flathub.org/login with a GitHub account, pressing "Developer Settings" and following the instructions there.

Please let me know when it's done, so I can remove the manual verification flag.

**Comment by RobLoach** [comment]
Thanks for highlighting this. Hope to get the token on there sometime soon.

**Comment by RobLoach** [comment]
@barthalion It's trying to verify `libretro.org` but it should be `libretro.com` . Know if there's a way to update the URL?

**Comment by barthalion** [comment]
Uhh, not without renaming the app. Has it always been `libretro.com`, or the domain has changed?

**Comment by barthalion** [comment]
That's too bad we accepted it with a wrong ID. I will mark it as verified manually, as changing ID isn't generally the best UX for regular users.

**Comment by RobLoach** [comment]
Is there a way to have it symlink or something? Or allow a different URL to verify against?

**Comment by barthalion** [comment]
Not really. I could probably do this and that directly in the database, but the manual verification is the use case for this. We use it for Firefox already and looks like this: https://beta.flathub.org/apps/org.mozilla.firefox
